cdk/script/export-monitoring-file

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+ROOT_DIR="${DIR}/../.."
+INDEX_JS_BUILT_FILE="${ROOT_DIR}/monitoring/dist/index.js"
+LAMBDA_REGIONS=("cmp-monitoring-lambda-eu-west-1" "cmp-monitoring-lambda-us-west-1" "cmp-monitoring-lambda-ap-southeast-2" "cmp-monitoring-lambda-ca-central-1")
+
+get_abs_filename() {
+  # $1 : relative filename
+  echo "$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
+}
+
+if test -f "${INDEX_JS_BUILT_FILE}"; # Check if index js file is in the monitoring file
+then
+	src_file=$(find $INDEX_JS_BUILT_FILE) # Gets the index.js file generated in the monitoring/dist. Throws error if it doesn't exist
+	absolute_src_file_path=$(get_abs_filename "$src_file") # Gets the absolute path of the index.js file
+	absolute_src_directory=$(dirname "$absolute_src_file_path") # Gets the src directory name the absolute_src_file_path
+	echo "----------------------------"
+
+	for region in "${LAMBDA_REGIONS[@]}"
+	do
+		echo "starting ${region}"
+		cd "${absolute_src_directory}" # Go to monitoring/dist folder
```

Comment:
```suggestion
		cd "./monitoring/dist" # Go to monitoring/dist folder
```

Feels like a lot of code is dedicated to finding absolute paths, wondering why not just use relative paths?

Reply:
I tried to do this but didn't quite work in Github action. I've abstracted away into a function and added more comments to make it clearer.